src/PositionService.Models.Dto/Requests/Position/Filters/FindPositionsFilter.cs

```diff
@@ -8,8 +8,8 @@ public record FindPositionsFilter : BaseFindFilter
     [FromQuery(Name = "isAscendingSort")]
     public bool? IsAscendingSort { get; set; }
 
-    [FromQuery(Name = "isActive")]
-    public bool? IsActive { get; set; }
+    [FromQuery(Name = "includeDeactivated")]
+    public bool IncludeDeactivated { get; set; }
```

Comment:
set 'false' by default

Reply:
bool default value is false. In case it's not initialized its value is false

Reply:
indeed. my bad